setup.py

```diff
@@ -24,6 +24,9 @@
         packages=setuptools.find_packages(exclude=['*checkup*']),
         include_package_data=True,
         package_data={'': [os.path.join('qcelemental', 'data', '*.json')]},
+        setup_requires=[
+            'pytest-runner'
+        ],
```

Comment:
Does this mean that `python setup.py install` will fail if pietist is not present?

Reply:
probably, https://github.com/pytest-dev/pytest/issues/3814 . :-(

Reply:
Can we do:
```
needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
pytest_runner = ['pytest-runner'] if needs_pytest else []

# ...

setup(
    #...
    setup_requires=[
        #... (other setup requirements)
    ] + pytest_runner,
)
```

as the suggested workaround?

Reply:
just pushed :-)